### ISSUE TITLE
training problem

### ISSUE THREAD

**Issue by JohnHerry** [lucidrains/e2-tts-pytorch#42]
training the project for about 200,000 steps but still got noise mel prediction  images. My batch size is 10 because of the memory of GPU is not large enough to make 32 sample batches. and I am using a kind of normalized mel spec prepared. which means each value of mel items is scaled to [-1, 1],   need suggestion.  is this model support normalized mel spec?
